src/assets/js/rtc.js

```diff
@@ -1,14 +1,17 @@
 /**
+ * @author Lorenzo Mangani, QXIP BV <lorenzo.mangani@gmail.com>
+ * @date 27th April, 2020
  * @author Amir Sanni <amirsanni@gmail.com>
  * @date 6th January, 2020
  */
 import h from "./helpers.js";
 import EventEmitter from "./emitter.js";
 import Presence from "./presence.js";
 import MetaData from "./metadata.js";
+
 var TIMEGAP = 6000;
 var allUsers = [];
-var enableHacks = false;
+var enableHacks = true;
```

Comment:
We want to have this on true?

Reply:
good point. the only effect is this allows clients without webcam/mic to join as receive-only. We can disable it.

Reply:
done!